manifest/service.go

```diff
@@ -82,11 +82,19 @@ func (s *Service) Proxies(app string) []Proxy {
 		if p.External() {
 			name := fmt.Sprintf("%s-%s-proxy-%d", app, s.Name, p.Balancer)
 
+			networkName := ""
```

Comment:
Maybe a short comment above this logic talking about its intent? Are we always getting the last network defined?